src/reference/asciidoc/amqp.adoc

```diff
@@ -996,6 +998,37 @@ For example, when running on a transacted listener container thread and performi
 In that case, the operations will be performed on that channel and committed when the thread returns to the container;
 it is not necessary to use `invoke` in that scenario.
 
+When using confirms in this way, much of the infrastructure set up for correlating confirms to requests is not really needed.
+Starting with _version 2.1_, the connection factory supports a new property `simplePublisherConfirms`.
+When this is `true` the infrastructure is avoided and the confirm processing will be more efficient.
+
+Furthermore, the `RabbitTemplate` sets the property `publisherSequenceNumber` in the sent message `MessageProperties`.
+If you wish to check (or log, etc) specific confirmations, you can do so with an overloaded `invoke` method:
+
+[source, java]
+----
+public <T> T invoke(OperationsCallback<T> action, com.rabbitmq.client.ConfirmCallback acks,
+        com.rabbitmq.client.ConfirmCallback nacks);
+----
+
+NOTE: These `ConfirmCallback` objects (for acks and nacks) are the Rabbit client callbacks, not the template callback.
```

Comment:
Why we really can't use there our own `org.springframework.amqp.rabbit.core.RabbitTemplate.ConfirmCallback` ?
It doesn't look too hard to delegate calls from the internal Rabbit client `ConfirmCallback` instances.

Reply:
It doesn't really fit; we don't have correlation data or a cause or a way to convey `multiple`.

I suppose I could subclass `CorrelationData` to provide the tag and `multiple`. WDYT?

Reply:
Ah! I see what you mean.
No, I think it's really not necessary to overhead here: since we don't have anything to wait for, therefore we don't need correlation.

So, the current solution is fine.

Will pull locally for deeper review and possible merging...